src/renderer/components/badge/badge.module.css

```diff
@@ -36,6 +36,18 @@
   text-overflow: ellipsis;
 }
 
+:global(.ItemListLayout) {
```

Comment:
Seems like this is wrong place for such styles and should be moved to `item-list-layout.scss`.
`Badge` component is more generic and probably shouldn't know about `ItemListLayout` at all.

Reply:
Fair enough, will investigate how to use a module css from within a non-module

Reply:
Or just convert `item-list-layout.scss` to css-module?

Reply:
That could work, thanks

Reply:
@ixrock Do you know now, even after converting to `css-module`, it would be possible to get this to work? Given that `badge.module.css` is its own local scope, I don't actually think that it is possible to do this without `.ItemListLayout` being global. 

I am having trouble finding documentation on how this might be done.